src/App.css

```diff
@@ -36,3 +36,15 @@
     transform: rotate(360deg);
   }
 }
+
+body {
+  overflow: hidden;
```

Comment:
bruh

test your shit.
How many years have you been in this program

Reply:
overflow hidden makes my life easier with the landing page stuff. Forgot to change when merging with yours. Will fix.